### PR TITLE
Preload all preview iframe font pairings

### PR DESCRIPTION
**Pull request by alshakero**
#### Changes proposed in this Pull Request

This PR proposes preloading all preview font pairings for a more seamless UX. People are likely to take a second and look around the page. It would be nice to have the fonts ready and get rid of the [FOUT](https://css-tricks.com/fout-foit-foft/) as shown here by p1589217382261500-slack-gutenboarding.

#### The challenging part

It's not enough to load the font in the browser cache. As even loading it from cache will still create a flash of fonts. It has to be in memory for it to appear. That's why this can't be solved by a service worker, `<link rel=preload"...` etc. I tried the preload way and the flash persisted. 

#### Testing instructions

Go to the font picker, switch between fonts and the fonts should change once per click. Not `Previous font -> default font -> new font`.

I tested in Chrome, FF, and Safari.

#### Downside

The whole preview page will not render until all the fonts are fetched.

### CODE REVIEW

File: client/landing/gutenboarding/onboarding-block/style-preview/preview.tsx [hunk 1]
```diff
@@ -16,14 +16,49 @@ import { isEnabled } from 'config';
...
+		return acc;
+	}, '' );
+
+	// Chrome doesn't load the fonts in memory until they're actually used,
+	// this keeps the fonts used and ready in memory.
+	const fontsHolders = fontPairings.reduce( ( acc, pairing ) => {
+		Object.values( pairing ).forEach( ( font ) => {
+			const fontHolder = document.createElement( 'div' );
+			fontHolder.style.fontFamily = font;
+			fontHolder.innerHTML = 'Placeholder';
```
Comment (ramonjd) [inline client/landing/gutenboarding/onboarding-block/style-preview/preview.tsx:46]: Is it necessary for the elements to contain text? Just curious.

Comment (alshakero) [reply client/landing/gutenboarding/onboarding-block/style-preview/preview.tsx:46]: Yes! I was amazed too!

**Comment by simison** [comment]
Would this close https://github.com/Automattic/wp-calypso/issues/40613 ?

**Comment by alshakero** [comment]
I would say yes. But I'll leave it for @sirreal to judge.

**Comment by sirreal** [comment]
> ~I would say yes. But I'll leave it for @sirreal to judge.~
> 
> Edit: Not really, I don't think it does.

I agree, I don't think this closes https://github.com/Automattic/wp-calypso/issues/40613. That's a good backlog task. The fonts implmentation feels unfinished, but maybe it's good enough 🤷‍♂️

**Comment by simison** [comment]
@alshakero did you want another review on this or is it ready to merge?

**Comment by alshakero** [comment]
It's good. I'll merge. Thanks!

**Pull request merged**